FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -13,12 +13,42 @@ public final class RemoteFeedLoader: FeedLoader {
 		case invalidData
 	}
 
+	public struct Root: Decodable {
```

Comment:
`Root` is a private implementation detail and should not be accessible from outside of the module

Please make `Root` private

```suggestion
	private struct Root: Decodable {
```